src/LSP12/addIssuedAssets/addIssuedAssets.test.ts

```diff
@@ -40,13 +40,15 @@ describe('addIssuedAssets', () => {
             'FirstTestToken',
             'FTT',
             owner.address,
+            0,
             true,
         );
 
         const secondIssuedAsset = await new LSP7Mintable__factory(owner).deploy(
             'SecondTestToken',
             'STT',
             owner.address,
+            0,
```

Comment:
Suggest using the constant from lsp-smart-contracts, or adding a comment

```suggestion
            LSP4_TOKEN_TYPES.TOKEN,
```